cli/upload.py

```diff
@@ -76,6 +76,17 @@ def _open_file_mapping(extra_metadata: dict, base_path: str) -> Dict[str, Binary
 
     open_files = {}
     for local_path, uuid in extra_metadata.items():
+
+        # if user wants us to get file from GCS
+        # and we want it to be analysed for extra_md
+        # we say we don't support it
+        if local_path.startswith("gs://"):
+            raise Exception(
+                "Google Storage files are not supported for this assay."
```

Comment:
I might phrase this differently. Maybe: "File transfers from Google Cloud Storage are not supported for this assay type. Please download locally the files that you wish to upload, update the file paths in your metadata Excel file, and try again."